mentat/code_edit_feedback.py

```diff
@@ -24,36 +23,47 @@ async def get_user_feedback_on_edits(
     match user_response.lower():
         case "y" | "":
             edits_to_apply = file_edits
-            # TODO: All of the 'user messages' here feel like they should be system messages;
-            # if we decide to change this, make sure to change the clear command as well
             conversation.add_message(
-                MessageRole.User, "User chose to apply all your changes."
+                {"role": "system", "content": "User chose to apply all your changes."}
```

Comment:
in these and the changes below we're changing these message roles from `user` to `system`. I believe they were originally `system`, and that feels like it makes more sense, but maybe that should be a separate change from this PR? Since it might impact model understanding, conversation viewer display, etc

Reply:
Hmm, ok. I do think they make a lot more sense to be system messages though, but I'll try benchmarking them.